components/TestimoniesTable/TestimoniesTable.js

```diff
@@ -15,14 +15,19 @@ const TestimonyRow = ({ testimony }) => {
   const senatorId = testimony.senatorId
   const representativeId = testimony.representativeId
 
+  const x = usePublicProfile(testimony.authorUid)
+  const authorPublic = x.result?.public
+  const loading = x.loading
+  const error = x.error
+
   return (
     <tr>
       <td>{testimony.billId}</td>
       <td>{testimony.position}</td>
       <td>
         {testimony.authorDisplayName == null ? (
```

Comment:
change `testimony.authorDisplayName == null` to `!testimony.authorDisplayName` to handle empty strings and undefined as well.